src/NpgsqlAnalyzers.Tests/RegressionTests.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using NpgsqlAnalyzers.Tests.Utils;
+using NUnit.Framework;
+using ThrowawayDb.Postgres;
+
+namespace NpgsqlAnalyzers.Tests
+{
+    [TestFixture]
+    public class RegressionTests : IDisposable
```

Comment:
You don't really need to make this thing `IDisposable` because you could just `using var db = ...` in the test method itself and that would be it instead of having to implement to a wrapping `IDisposable` here for the test fixture.

Reply:
Yep, nice one. Will simplify. Future regression additions can be added to the existing test method, as the whole purpose of this class is a test encompasses the whole DB and does not fail.